documentation/man/README.md

```diff
@@ -284,3 +286,4 @@ AUTHOR
 Copyright (C) 2002-2024, GRAME - Centre National de Creation Musicale.
 All rights reserved.
 
+make[1]: Leaving directory '/home/olaf/projects/rust/forks/faust/documentation/man'
```

Comment:
Committing this line looks a bit like an accident?

Reply:
I guess i should not run this make step myself i guess it should be done in the versiinig step